pkg/resources/kafka/poddisruptionbudget.go

```diff
@@ -62,6 +66,21 @@ func (r *Reconciler) podDisruptionBudget(log logr.Logger) (runtime.Object, error
 // Calculate maxUnavailable as max between brokerCount - 1 (so we only allow 1 broker to be disrupted)
 // and 1 (to cover for 1 broker clusters)
 func (r *Reconciler) computeMinAvailable(log logr.Logger) (intstr.IntOrString, error) {
+	podList := &corev1.PodList{}
+	matchingLabels := client.MatchingLabels(kafka.LabelsForKafka(r.KafkaCluster.Name))
+	err := r.Client.List(context.TODO(), podList, client.ListOption(client.InNamespace(r.KafkaCluster.Namespace)), client.ListOption(matchingLabels))
+	if err != nil {
+		return intstr.FromInt(-1), errors.WrapIf(err, "failed to reconcile resource")
+	}
+
+	// number of healthy brokers in the KafkaCluster
+	brokers := 0
+	for _, pod := range podList.Items {
+		if pod.Status.Phase == corev1.PodRunning {
+			brokers++
+		}
+	}
+
 	/*
```

Comment:
this inline topic was meant to be at the top of the method body. let's keep it there? or move it as a godoc

Reply:
done!